doc/docs/tutorials/try_controller.md

```diff
@@ -1,36 +1,55 @@
 # Try the Pipelines Controller with HAPI FHIR Store
```

Comment:
This file is not linked from anywhere and does not appear in the site navigation, correct? I am not sure if it is actually needed as separate doc; we should probably fold its content into the "single machine" or deploy controller" docs.